js/plugins/google-genai/src/common/types.ts

```diff
@@ -119,7 +119,7 @@ export enum DynamicRetrievalMode {
 /**
  * Schema for parameters passed to {@link FunctionDeclaration.parameters}.
  */
-export interface FunctionDeclarationSchema {
+export declare interface FunctionDeclarationSchema {
```

Comment:
you actually don't need `declare` if you're importing as `type` (which I see that you do as well)

Reply:
oh, you don't... just update imports to `import { type FunctionDeclarationSchema } from '...'`